### PR TITLE
Dominik's template update

### PR DESCRIPTION
**Pull request by Gldkslfmsd**
It's working and it's documented. Added: Makefile, e.g. For list of updates see Chapter 3.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+# defense_slides/slides.pdf: defense_slides/slides.tex
+# 		cd defense_slides; latexmk  -pdflatex="xelatex %O %S" -pdf -dvi- -ps- slides
+
+
+# simple continuous integration: git pull & commit & push
+.PHONY: ci
+ci:
+	# make sure git will remember my password
+	git config credential.helper store
```
Comment (jindrahelcl) [inline Makefile]: this line should not be here for various reasons - I might use ssh keys (which I do), I might not want git to remember my password, or generally I don't want makefiles to go about changing my git configuration. If people want to store git password, they should set the config themselves, manually .

File: README.md [hunk 2]
```diff
@@ -6,12 +6,7 @@ The font is Linux Libertine and it needs to be installed first (depending on the
 Compile with:
 
 ```
-cd thesis
-lualatex thesis
-bibtex thesis
-makeglossaries thesis
-lualatex thesis
-lualatex thesis
+make
```
Comment (jindrahelcl) [inline README.md]: I would leave the second option in the README for those who don't want to use make (and the implied latexmk) (for me it was just too painful to set up and I would run all those commands semi-automatically from emacs).

File: README.md [hunk 3]
```diff
@@ -21,3 +16,52 @@ pigz -d anthology.bib.gz
...
+
+### How to send feedback:
+
+- The author will be very thankful for feedback during writing. Most preferrable ways:
+
+1) comments or edit suggestions in Latex macros, e.g.
+ - \XXX{YourName: your comment}
+ - \repl{to be replaced}{by this}    or   \repl{}{insert this here}   or   \repl{delete this}{}
+ - you can add macros for your comments to macros.tex , e.g. \def\Raj#1{\XXX{Raj: #1}}
+
```
Comment (jindrahelcl) [inline README.md:74]: I personally prefer using 
```\def\JH#1{{\color{blue} JH: \emph{ #1}}}```
because then it is clear who commented on what. Collaborators can define their macros using their initials and their favorite color.

File: acronyms.tex [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+%\newacronym{pi}{PI}{path integral}
+%\newacronym{qa}{QA}{question answering}
+%\newacronym{rbf}{RBF}{radial basis function}
+%\newacronym{relu}{ReLU}{rectified linear unit}
+%\newacronym{rl}{RL}{reinforcement learning}
+%\newacronym{rouge}{ROUGE}{Recall-Oriented Understudy for Gisting Evaluation}
+%\newacronym{rnn}{RNN}{recurrent neural network}
+%\newacronym{sgd}{SGD}{stochastic gradient descent}
+%\newacronym{smt}{SMT}{statistical machine translation}
+%\newacronym{sota}{SoTA}{state-of-the-art}
```
Comment (jindrahelcl) [inline acronyms.tex:75]: don't use this unless you want to make a bad impression on someone like me :)

File: Makefile (hunk 1 above)
Comment (Gldkslfmsd) [reply Makefile]: ok

File: README.md (hunk 2 above)
Comment (Gldkslfmsd) [reply README.md]: ok

File: README.md (hunk 3 above)
Comment (Gldkslfmsd) [reply README.md:74]: ok. It's a template, everybody can edit it as they prefer

File: acronyms.tex (hunk 4 above)
Comment (Gldkslfmsd) [reply acronyms.tex:75]: it's commented. It's users' responsibility
